doc/api/v8.md

```diff
@@ -1304,6 +1304,37 @@ setTimeout(() => {
 }, 1000);
 ```
 
+## `v8.isStringOneByteRepresentation(content)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `content` {string}
+* Returns: {boolean}
+
+V8 only supports `Latin-1/ISO-8859-1` and `UTF16` as the underlying representations.
+If the `content` uses `Latin-1/ISO-8859-1` as the underlying representation, this function will return true;
+otherwise, it returns false.
+
+This method returns false does not mean that the string contains some characters not in `Latin-1/ISO-8859-1`.
+Sometimes a `Latin-1` string may also be represented as `UTF16`.
+
+```js
+const { isStringOneByteRepresentation } = require('node:v8');
+const bf = Buffer.alloc(20);
+
+function writeString(input) {
+  if (isStringOneByteRepresentation(input)) {
+    bf.write(input, 'latin1');
+  } else {
+    bf.write(input, 'utf16le');
+  }
```

Comment:
This isn't an example that would ever show up in real-world code, since there would be no way to look at the buffer and know what encoding was used to create its contents -- what is this telling users?

Reply:
Thank you for pointing that out. I've learned a lot about how to write good documentation.